TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/Utilities.java

```diff
@@ -0,0 +1,101 @@
+package org.firstinspires.ftc.teamcode.teleop;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+import org.firstinspires.ftc.teamcode.movement.MoveRobotLinear;
+
+
+/**
+ *      Class defining all required actions for a controlled stage.
+ *
+ *      Supports
+ *          * Moving the robot on all 4 axes.
+ *          * Moving the mineral arm.
+ *          * Extending/ Contracting the robot arm.
+ *          * Spinning the robot.
+ *          * Changing the direction of the mineral brush.
+ *          * Tilting the mineral box at 90/ 45 degrees from the mineral arm.
+ *          * Turning the correction on/ off.
+ */
+
+public class Utilities {
+
+    public static void moveForward(MoveRobotLinear movementController, Gamepad gamepad, LinearOpMode opMode) {
+        while(opMode.opModeIsActive() && gamepad.dpad_up) {
```

Comment:
Ce ar fi daca ai lua valorile din `gamepad.left_trigger` si `gamepad.right_trigger` in cate o variabila? In felul asta, nu mai accesezi de 2 ori.